src/codegen/function_body_chunk.rs

```diff
@@ -339,7 +318,7 @@ impl Builder {
         } else {
             Chunk::FromGlibConversion {
                 mode: parameter.into(),
-                array_length: parameter.array_length.clone(),
+                array_length_name: self.find_array_length_name(&parameter.name),
```

Comment:
@sdroege In gtk-rs no functions that uses this (out array with out length), you have functions to test it?